### DISCUSSION TITLE
Adding maps of solar system planets?

### DISCUSSION THREAD

**Discussion by shanebyrneonmars** [Stellarium/stellarium#3772] [Q&A]
Hello, I'm new to Stellarium but work a lot with NASA images. I'd like to add custom maps of planets, can this be done? I'd also like to display lat/lon grids and a terminator line when zoomed up on a solar system body. Thanks!

**Comment by gzotti** [comment]
There is scripting support for alternative textures: http://stellarium.org/doc/24.0/classSolarSystem.html#a2f831a3888fbd6e0f3828b7036e1d62b

There is no grid.

Terminator comes automatically from solar illumination, but not with an additional line.

**Comment by shanebyrneonmars** [comment]
Thanks, I was able to get the view I needed through scripting and swapped out the texture map.  I just added the lat/lon lines to the new texture map as a workaround.
